pages/linux/httpie.md

```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
+
+- Send a GET request (default method with no request data):
+
+`http {{https://api.github.com/users}}`
+
+- Send a POST request (default method with request data):
+
+`http {{example.org}} {{hello=World}}`
+
+- Send a POST request with redirected input:
+
+`http {{example.org}} < {{file.json}}`
+
+- Send a PUT request:
```

Comment:
Let us remove the request header value here and put it in the DELETE example. And change the description to `Send a PUT request with a given json body`

Reply:
I've changed descriptions. PTAL.